auction-server/build.rs

```diff
@@ -1,13 +1,23 @@
-use std::process::Command;
+use {
+    anchor_lang_idl::{
+        convert::convert_idl,
+        types::{
+            Idl,
+            IdlInstructionAccountItem,
+        },
+    },
+    std::{
+        fs,
+        process::Command,
+    },
+};
 
-fn main() {
+fn build_evm_contracts() {
     let contract_setup = r#"
         cd ../contracts/evm
         forge build --via-ir
     "#;
     println!("cargo:rerun-if-changed=../contracts/evm");
-    println!("cargo:rerun-if-changed=migrations");
```

Comment:
why is this removed? I added this because of this:
https://docs.rs/sqlx/latest/sqlx/macro.migrate.html

Reply:
It isn't removed. It's at the beginning of the `main` function

Reply:
my bad